services/create_file.rb

```diff
@@ -1,18 +1,21 @@
+require_relative '../translation'
+
 class CreateFile
+  include ::Translation
```

Comment:
Now you can add [template method pattern](https://medium.com/@joshsaintjacque/the-template-method-pattern-558f3e16879f). That will include your translation module only once.
```ruby
class AbstractFile
  include ::Translation

  def call
    raise NotImplemented, "you should define #call method"
  end

  private

  def check_file_presence(file_path)
     # you have the same code chunk in 3 different places
    unless File.exist?(file_path)
      translate(:file_not_found)
      return
    end
  end
end

class CreateFile < AbstractFile
  # no need to include anything
  # and now you have a guarantee that all classes used within a Factory have the same interface
end
```

Reply:
Created AbstractService